docs/release-notes/0.11/release-0.11.md

```diff
@@ -0,0 +1,81 @@
+# .NET for Apache Spark 0.11 Release Notes
+
+### New Features and Improvements
+
+* Ability to pass and return corefxlab DataFrames to UDF APIs ([#277](https://github.com/dotnet/spark/pull/277))
+* Supporting ML TF-IDF (Term frequency-inverse document frequency) feature vectorization method ([#394](https://github.com/dotnet/spark/pull/394))
+* Support for TimestampType in `DataFrame.Collect()`, `CreateDataFrame` and UDFs ([#428](https://github.com/dotnet/spark/pull/428))
+* Support for Broadcast Variables ([#414](https://github.com/dotnet/spark/pull/414))
+* Implement ML feature Word2Vec ([#491](https://github.com/dotnet/spark/pull/491))
+* Streamline logging when there is a failure ([#439](https://github.com/dotnet/spark/pull/439))
+
+
+### Breaking Changes
+
+* SparkSession.Catalog call changed from a method to a property ([#508](https://github.com/dotnet/spark/pull/508))
+
+### Compatibility
+
+<table>
+    <thead>
+        <tr>
+            <th>Oldest compatible .NET for Apache Spark worker</th>
```

Comment:
Can you describe what this table means? I have one column with "compatible" and another with "incompatible" so it's a bit confusing.

Reply:
yep makes sense, added a description. Please let me know if that makes sense

Reply:
Can we also put a link to PR (or tests) for each incompatible feature?

Reply:
Do we prefer PR or test?

Reply:
Added links to PR